src/web-page.ts

```diff
@@ -1,7 +1,7 @@
 export class WebPage {
 
     public protocol : string;
-    public headers : object;
+    public headers : any;
```

Comment:
Why change that to `any`? Isn't the structure of the headers an object with a structure like this `{ [ property : string ] : string }`?